.circleci/config.yml

```diff
@@ -19,12 +19,12 @@ jobs:
     steps:
       - checkout
       - restore_cache:
-          key: rails-demo-{{ checksum "Gemfile.lock" }}
+          key: codeherd-{{ checksum "Gemfile.lock" }}
```

Comment:
is this fixing the deploy issue?

Reply:
no, its me trying to figure out how cacheing works in CircleCI. Also is there a deploy issue that I'm not aware of?

Reply:
I thought you had said that it wasn't set up for automatic deploy.